stubs/mu-plugin.php

```diff
@@ -0,0 +1,144 @@
+<?php
+/*
+ * Plugin Name: CacheWerk k6 (MU)
+ * Plugin URI: https://github.com/cachewerk/k6
+ * Description: ...
+ * Version: 1.0.0
+ * Author: Rhubarb Group
+ * Author URI: https://rhubarb.group
+ * Requires PHP: 7.2
+ */
+
+defined('ABSPATH') || exit;
+
+add_action('muplugins_loaded', ['k6ObjectCacheMetrics', 'init']);
+add_action('shutdown', ['k6ObjectCacheMetrics', 'print'], PHP_INT_MAX);
```

Comment:
Let's do the same logic as Object Cache Pro to be safe. This will avoid printing when it's not a HTML request.

```php
add_action('wp_footer', [$this, 'shouldPrintMetricsComment']);
add_action('wp_body_open', [$this, 'shouldPrintMetricsComment']);
add_action('login_head', [$this, 'shouldPrintMetricsComment']);
add_action('in_admin_header', [$this, 'shouldPrintMetricsComment']);
add_action('rss_tag_pre', [$this, 'shouldPrintMetricsComment']);

add_action('shutdown', [$this, 'maybePrintMetricsComment'], PHP_INT_MAX);
```